docs/yarn.lock

```diff
@@ -7360,9 +7360,9 @@ react-json-view-lite@^1.2.0:
   integrity sha512-Itc0g86fytOmKZoIoJyGgvNqohWSbh3NXIKNgH6W6FT9PC1ck4xas1tT3Rr/b3UlFXyA9Jjaw9QSXdZy2JwGMQ==
 
 react-live@^4.1.5:
-  version "4.1.5"
-  resolved "https://registry.yarnpkg.com/react-live/-/react-live-4.1.5.tgz#a4fa4cfdcad763503a209a29bace3339764fdfb1"
-  integrity sha512-ul3Zwvqvh6KTg8j7xGCT26+c8J9vQ+LFUrZCbrrrzEExuVB/39s1GKG3NsywnL+aGAjpfnUTaVCe7KlKIvVPiw==
```

Comment:
Thanks for your contributions, 
yarn lock changes should be reverted as well , as changes in this file are done by internal team only.

Reply:
Oh ok, I'm on it

Reply:
@jdevcs I've revert it and push again. can you please check it.